coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -5396,6 +5396,10 @@ def clamp(context, node):
         context.add(mb.clip(x=x, alpha=min_val, beta=max_val, name=node.name))
 
 
+@register_torch_op
```

Comment:
Rather than create a new function. It would be better to create an alias. Change the `@register_torch_op` line for `clamp` (i.e line 5375) to `@register_torch_op(torch_alias=["clip"])`.